cpp/src/strings/capitalize.cu

```diff
@@ -173,12 +178,15 @@ std::unique_ptr<column> capitalize_utility(CapitalFn cfn,
 }  // namespace
 
 std::unique_ptr<column> capitalize(strings_column_view const& input,
+                                   string_scalar const& delimiters,
                                    rmm::cuda_stream_view stream,
                                    rmm::mr::device_memory_resource* mr)
 {
+  CUDF_EXPECTS(delimiters.is_valid(stream), "Delimiter must be a valid string");
   if (input.is_empty()) return make_empty_column(data_type{type_id::STRING});
-  auto d_column = column_device_view::create(input.parent(), stream);
-  return capitalize_utility(capitalize_fn{*d_column}, input, stream, mr);
+  auto const d_column     = column_device_view::create(input.parent(), stream);
+  auto const d_delimiters = delimiters.value(stream);
+  return capitalize_utility(capitalize_fn{*d_column, d_delimiters}, input, stream, mr);
```

Comment:
I think a function named `*_utility` should not return anything. `utility` should be preserved for namespace, as it does not contain any explicit meaning about what it is doing. Thus, we should rename it to something else.

Reply:
I've never heard of this rule. I can rename it to something else.